cloudant-client/src/main/java/com/cloudant/client/api/model/ReplicatorDocument.java

```diff
@@ -227,12 +229,16 @@ public void setRetriesPerRequest(Integer retriesPerRequest) {
         replicatorDocument.setRetriesPerRequest(retriesPerRequest);
     }
 
-    public Integer getSinceSeq() {
+    public JsonElement getSinceSeq() {

```

Comment:
To avoid breaking compatibility with anyone using this still with e.g. Couch 1.x we should not change this API.
I suggest reverting this and adding:
`String getSinceSeqString()` - which can output the JsonElement string (for both string and number seq types), the existing int method can be modified to check if the JsonElement is a number primitive and return the int if it is and throw (I guess an `IllegalStateException`) if it isn't.

Reply:
Fixed in 103eb3b and 0c4e3c8